_posts/2019-08-10-PC-Engines-Firmware-v4-10-0-0.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.10.0.0"
+date:   2019-08-10
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.10.0.0
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** 2a20d13.
+2. Added ACPI support for GPIOs. Detailes are available in
+[apu2-documentation](https://github.com/pcengines/apu2-documentation/blob/master/docs/gpios.md).
+So far access to GPIOs was possible only via dedicated driver. Now, there is
+support via kernel sysfs including:
+    - LEDs
+    - S1 switch button
+    - SIMSWAP
+
```

Comment:
@piotr-kleins we did a lot more:
- write about ESXi validation that we are performing. We have located the issue and are working to resolve it.
- added documentation about CPU boost: https://github.com/pcengines/apu2-documentation/blob/master/docs/apu_CPU_boost.md
- updated WLE200NX documentation with issue workarounds: https://github.com/pcengines/apu2-documentation/blob/master/docs/mpcie_modules.md#wle200nx